packages/common/infra/src/op/message.ts

```diff
@@ -150,6 +151,7 @@ export abstract class AutoMessageHandler {
 
   close() {
     this.port.close?.();
+    this.port.terminate?.(); // For Worker
```

Comment:
lifecycle management for worker or channel should be kept outside

Reply:
[Worker](https://developer.mozilla.org/en-US/docs/Web/API/Worker/terminate) 
Worker is somewhat different from SharedWorker and MessagePort. There is no close method.

Reply:
It should be possible to move it out

Reply:
move worker termination to where it was created